src/services/SpeciesService.ts

```diff
@@ -404,6 +406,19 @@ const getSpeciesProjects = (
   return SearchService.search<SpeciesProjectsSearchResponse>(params);
 };
 
+const mergeOtherSpecies = async (
+  mergeOtherSpeciesPayload: MergeOtherSpeciesPayload,
+  observationId: number
+): Promise<Response> => {
+  const urlReplacements = {
+    '{observationId}': `${observationId}`,
+  };
+  return await httpMergeOtherSpecies.post({
```

Comment:
If you aren't modifying the result, or attempting to catch any errors that occur with the `post`, it is not necessary to mark this function as `async` and use `await` within it.